hedera-node/hapi-fees/src/main/java/com/hedera/node/app/hapi/fees/usage/token/TokenGetInfoUsage.java

```diff
@@ -85,4 +90,9 @@ public TokenGetInfoUsage givenCurrentlyUsingAutoRenewAccount() {
         addRb(BASIC_ENTITY_ID_SIZE);
         return this;
     }
+
+    public TokenGetInfoUsage givenCurrentMetadata(final String metadata) {
+        addRb(metadata.length());
```

Comment:
Do we want to add a comment here explaining why this is so different than the fees for other messages? Just a thought

Reply:
@mhess-swl Is there something specific you are missing? I am afraid properly documenting the fee system is a rabbit hole. We should revisit once modularization is done.

Reply:
I'm referring to the fact that the pricing model for this endpoint is significantly different than the pricing models in other endpoints